registry/adapters/kubernetes/kubernetes.go

```diff
@@ -107,8 +213,8 @@ func (a *Adapter) ListInstances() ([]*api.ServiceInstance, error) {
 
 // ListServiceInstances queries for the list of service instances currently registered for the given service.
 func (a *Adapter) ListServiceInstances(serviceName string) ([]*api.ServiceInstance, error) {
```

Comment:
One list all instances, and the other one list instances for a particular service.
This is part of the [`api.ServiceDiscovery`](https://github.com/amalgam8/amalgam8/blob/master/registry/api/types.go#L24) interface, so required to be here in order to implement it.